newdle_frontend.js

```diff
@@ -60,9 +72,20 @@ function checkGuess() {
     }
 }
 
-
 function winningConditions(){
     if (document.getElementById("presented_headline").innerText === headlineArray.join(" ")){
-        document.getElementById("congratulations_text").innerText = "🎊📰 Congratulations! 📰🎊"
+        document.getElementById("congratulations_text").innerText = `🎊📰 Congratulations! 📰🎊 \n Time: ${time_translator(zeroth_millisec, final_millisec = new Date().getTime())} minutes`
```

Comment:
why do you name the second argument final_millisec if you are using it directly in the call?
I would still do this but define it as a const before setting the value of the congratulations text